### PR TITLE
Add fast to computed version digest

### PR DESCRIPTION
**Pull request by katrogan**
# TL;DR
Add fast to computed version digest. This helps to identify whether versions of code are fast-registered.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
https://docs.google.com/document/d/1n7DmKXaOv8dAb9eHD4TKqka6FsdJNs3Tugpf1I6nvMg/edit?usp=sharing

## Tracking Issue
https://github.com/lyft/flyte/issues/554

## Follow-up issue
_NA_

### CODE REVIEW

**Comment by kumare3** [comment]
ohh i did not mean this, I meant to the "version" in the registered entity

**Comment by kumare3** [comment]
Looks good to me. (I think in the docs we should mention that we only copy *.py files, and not the config files)

**Review by kumare3 — DISMISSED** [review]

File: flytekit/tools/fast_registration.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def compute_digest(source_dir: _os.PathLike) -> str:
     :param _os.PathLike source_dir:
     :return Text:
     """
-    return _dirhash.dirhash(source_dir, "md5", match=["*.py"])
+    return "fast{}".format(_dirhash.dirhash(source_dir, "md5", match=["*.py"]))
```
Comment (wild-endeavor) [inline flytekit/tools/fast_registration.py:21]:
```suggestion
    return f"fast{_dirhash.dirhash(source_dir, "md5", match=["*.py"]}")
```

Comment (wild-endeavor) [reply flytekit/tools/fast_registration.py:21]: up to you.

**Review by wild-endeavor — APPROVED** [review]

**Review by wild-endeavor — APPROVED** [review]

**Pull request merged**